protocol/signalfx/signalfxlistener.go

```diff
@@ -294,10 +290,7 @@ func SetupChain(ctx context.Context, sink Sink, chainType string, getReader func
 	}
 	st := &sfxclient.WithDimensions{
 		Collector: sfxclient.NewMultiCollector(
-			&metricTracking,
-			&errorTracker,
 			counter,
```

Comment:
Just making sure if we are okay with deleting these metrics?

Reply:
No this is not what we want to do.  We want to exclude the counter from this multi collector, not delete the contents and keep just the counter.  Explore that code and figure out what is actually going on, then come to me and we can discuss it.